substrate/primitives/crypto/ec-utils/src/lib.rs

```diff
@@ -15,272 +15,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Elliptic Curves host functions which may be used to handle some of the *Arkworks*
-//! computationally expensive operations.
+//! Elliptic curves which are mostly compatible with *Arkworks* library
+//! mostly useful in non-native contexts.
```

Comment:
maybe?
```suggestion
//! This crate offers elliptic curves functionality which is mostly compatible with *Arkworks* library.
//! It is primarily designed for use in non-native contexts.
```

also wouid be nice to have link to Arkworks.